### DISCUSSION TITLE
Add FATAL check status

### DISCUSSION THREAD

**Discussion by m4rc1e** [fonttools/fontbakery#4364] [Ideas]
I'd like a new check status which is even more severe than FAIL. It should be named something like FATAL! This status should be used for checks which are essential for a family to successfully get onboarded to Google Fonts such as metadata files parsing correctly.

Implementing this check status will allow me to finally set the google/fonts ci to fail. I don't want to raise error codes on just FAILs since they are too strict and will fail many older families.

Discussion triggered by https://github.com/google/fonts/issues/7078

cc @chrissimpkins

**Comment by felipesanches** [comment]
"I understand the pain, but I disagree on the prescribed medicine."

By hypothetically creating a FATAL code, we would be inviting FAILs to be more broadly ignored.
Just like nowadays WARNs can be regularly ignored.

Also, at some point FATALs would also affect old font families. So, we'r eprone to face the same problem again in the short term.

In my opinion, a better solution would be to put in place mechanisms for documenting exceptions for eventual FAILs, and making the CI job ignore those FAILs that have explicit excuses documented.

**Comment by m4rc1e** [comment]
Did we find a solution to this issue? imo it's a quite important to solve.

**Comment by vv-monsalve** [comment]
Based on the chat conversation where this option was initially discussed, the FATAL log-level result would be related to issues that would break the pipeline workflow, not directly or necessarily to font binaries. Similar to the `ERROR` log level that is specifically related to the tool's issues or bugs. 

So, it could work like this:

`FATAL` for things that would break the pipeline workflow (like a typo in a METADATA.pb file).
`ERROR` for things related to FB itself.
`FAIL` for font related issues that must be addresed.

**Comment by felipesanches** [comment]
OK. I will make an initial implementation. Could you please list here which checks would be the first ones to be changed from `FAIL` to `FATAL`? I'll use this initial list of check-IDs in my implementation.

**Comment by vv-monsalve** [comment]
The last one that initiated this [com.google.fonts/check/metadata/parses](https://font-bakery.readthedocs.io/en/stable/fontbakery/profiles/googlefonts.html#com.google.fonts/check/metadata/parses)) reported [here](https://github.com/google/fonts/pull/7074#issuecomment-1843544687).

**Comment by simoncozens** [comment]
We did this.
